projects/ngx-sub-form/src/lib/helpers.ts

```diff
@@ -20,8 +20,7 @@ import {
   OneOfControlsTypes,
   TypedFormGroup,
 } from './shared/ngx-sub-form-utils';
-
-export const deepCopy = <T>(value: T): T => JSON.parse(JSON.stringify(value));
+import { cloneDeep } from 'lodash';
```

Comment:
```suggestion
import { cloneDeep } from 'lodash-es';
```

we should never import from lodash directly otherwise it'll pull all of lodash into ngx sub form, that'd make quite a huge diff on the bundle size